packages/mobile/src/app/WebViewScreen.tsx

```diff
@@ -3,33 +3,54 @@ import { StackScreenProps } from '@react-navigation/stack'
 import * as React from 'react'
 import { StyleSheet, View } from 'react-native'
 import { WebView } from 'react-native-webview'
+import { ShouldStartLoadRequest } from 'react-native-webview/lib/WebViewTypes'
```

Comment:
nit: No way to combine the 2 imports?

Reply:
I'm not sure how, VSCode added them this way, I didn't do it manually.